### PR TITLE
Core: Fill in context and selector properties for .live() support

### LINKED ISSUE #79: Move selector and context properties to Migrate for `.live()` support
See https://github.com/jquery/jquery/issues/1908

### PR DESCRIPTION
**Pull request by dmethvin**
Fixes #79

Ref https://github.com/jquery/jquery/issues/1908

### CODE REVIEW

File: src/traversing.js [hunk 1]
```diff
@@ -4,3 +4,11 @@ jQuery.fn.andSelf = function() {
 	migrateWarn("jQuery.fn.andSelf() replaced by jQuery.fn.addBack()");
 	return oldSelf.apply( this, arguments );
 };
+
+var oldFind = jQuery.fn.find;
```
Comment (dmethvin) [inline src/traversing.js]: Needs to be moved to the top of the file.

File: test/core.js [hunk 2]
```diff
@@ -47,6 +47,49 @@ test( "jQuery(html) loose rules", function() {
...
+	expect( 18 );
+
+	var test;
+
+	test = jQuery( undefined );
+	equal( test.selector, "", "Empty jQuery Selector" );
+	equal( test.context, undefined, "Empty jQuery Context" );
+
+	test = jQuery( document );
+	equal( test.selector, "", "Document Selector" );
```
Comment (dmethvin) [inline test/core.js]: I ran the jquery-migrate unit tests against the core PR built locally, by using the "dev" option in the jquery-migrate unit test dropdown. It seemed to pass so we should be good. We should actually land this PR first I'd say, to be sure that it works correctly with older core versions on all supported browsers.

Comment (mzgol) [inline test/core.js]: OK, I just wanted to get it tested properly. Since you checked it on your own, I'm good. :)

**Comment by dmethvin** [comment]
Landed at 683b9b3ae40e3fe9e83f3dd39d8e5e958975f3fa

**Pull request closed without merging**